perfkitbenchmarker/linux_packages/ycsb.py

```diff
@@ -137,7 +137,7 @@ def _GetWorkloadFileList():
   if FLAGS.ycsb_workload_files:
     return FLAGS.ycsb_workload_files
   return [data.ResourcePath(os.path.join('ycsb', workload))
-          for workload in ('workloada', 'workloadb')]
```

Comment:
Do we want to run it by default? Or just include it for people who want to run it?